tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es6",
+    "target": "es2018",
```

Comment:
why this change?

Reply:
The async iterable stuff comes with typescript es2018, without it the tests won't recognize it's keywords